eox_tenant/filters/pipeline.py

```diff
@@ -57,3 +61,61 @@ def run_filter(self, context, custom_template, *args, **kwargs):  # pylint: disa
         raise CertificateRenderStarted.RenderAlternativeInvalidCertificate(
             "You can't generate a certificate from this site.",
         )
+
+
+class OrgAwareLMSURLStudio(PipelineStep):
+    """
+    Filter tenant aware links from Studio.
+    """
```

Comment:
Same comment here about explicitly explaining what these do. Consider also this comment for line 94 and for the unittests docstrings.